src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php

```diff
@@ -71,8 +84,13 @@
 /**
  * Class CategoryController is responsible for "Sell > Catalog > Categories" page.
  */
-class CategoryController extends FrameworkBundleAdminController
+class CategoryController extends PrestaShopAdminController
 {
+    public function __construct(
+        private readonly MultistoreFeature $multistoreFeature,
```

Comment:
Maybe this would be a good candidate to be added in `PrestaShopAdminController` subscribed services Like we did for the context services